src/lib/resources.ts

```diff
@@ -270,4 +275,27 @@ export class Items extends OPResource {
 
         return ObjectSerializer.deserialize(data, "Array<ItemFile>");
     }
+
+
+    /**
+     * Get Item's OTP.
+     * itemQuery param can be an item's Title or ID.
+     *
+     * If there are more than one OTP field in an item
+     * it always returns the first/main one.
```

Comment:
I'd add this part of the documentation for the `getItemOTP` as well.

Reply:
I thought about that, but as our model classes are automatically generated, not sure if that addition won't break them.
Let me check that.
Overall I'm ok with adding it inside model class.

Reply:
We no longer want to be constrained by the generation tool. We used it at the beginning to have a decent starting point. Now we want to make this SDK on its own and, therefore, no longer rely on the openAPI generation tool.

Therefore, you can add it without any concerns 😄